src/foam/mlang/mlang.js

```diff
@@ -1724,6 +1724,7 @@ foam.CLASS({
       name: 'arg1'
     },
     {
+      class: 'Long',
```

Comment:
This isn't going to work with Double or Float properties. Better to make it a double.